src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1014 @@
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_testcase
+from util import configurable_testcase, writeConfig, loadConfig
+
+
+class FederalGovernmentDatabaseUpdateTestcase(sas_testcase.SasTestCase):
+  """SAS federal government database update test cases"""
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_FDB_1_default_config(self, filename):
+    """Generates the WinnForum configuration for FDB.1"""
+
+    # Load device info
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Load grant request.
+    grant_g1 = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the exclusion zone 'Yuma Proving Ground'
+    #  frequency range 'F1' which is .
+    grant_g1['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3650000000,
+        'highFrequency': 3660000000
+    }
+    grant_g2 = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the exclusion zone
+    # 'Yuma Proving Ground' frequency range 'F1' which is.
+    grant_g2['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3675000000,
+        'highFrequency': 3685000000
+    }
+    grant_g3 = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the modified
+    # exclusion zone 'Yuma Proving Ground' frequency range 'F2' which is.
+    grant_g3['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3575000000,
+        'highFrequency': 3585000000
+    }
+
+    # Update the location 'X' of CBSD contained witin the exclusion zone
+    # 'Yuma Proving Ground'.
+    device_b['installationParam']['latitude'] = 32.94414
+    device_b['installationParam']['longitude'] = -113.85681
+
+    # Creating conditionals for Cat B devices
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+    del device_b['measCapability']
+
+    conditionals = [conditionals_b]
+
+    # Create the actual config.
+    config = {
+        'registrationRequest': [device_b],
+        'grantRequests': [grant_g1, grant_g2, grant_g3],
+        'conditionalRegistrationData': conditionals
+      }
```

Comment:
Please add a TODO to include the database configuration.

Reply:
Updated   as per the comment, added TODO for database configurations